test/parallel/test-benchmark-zlib.js

```diff
@@ -9,5 +9,7 @@ runBenchmark('zlib',
                'method=deflate',
                'n=1',
                'options=true',
-               'type=Deflate'
+               'type=Deflate',
+               'inputLen=1024',
+               'duration=1'
```

Comment:
I'm -0 on this. It's heuristic (so still might fail on a really slow or clogged machine), probably safer to call `bench.end(gb)` with 
```js
// Give result in GBit/s, like the net benchmarks do.
// Never return 0, so that tests won't fail on really slow machines.
const gb = (readFromOutput * 8 / (1024 ** 3)) || 1
```

Reply:
ohh and https://github.com/nodejs/node/issues/20128 - settings `duration=1` will add a 1sec test to the suite.

Reply:
Do we do this elsewhere? Otherwise it seems like it would falsify test results – what if the test is *actually* broken and returns 0?

Would you be be okay with the current code if it passes a stress test run?

Reply:
Regarding the duration: Yeah, that’s not great. It’s not quite as bad as if it were a `sequential` test though…

Reply:
Let me read through the test and harness a little bit more. AFAIK the `test-benchmark-*` set are there "mostly" for syntax, API and regression testing. Designing correctness tests for the benchmarks might be out-of-scope.
Maybe we can just relax the `Error: called end() with operation count <= 0` rule anyway 🤔

Reply:
Found it:
https://github.com/nodejs/node/blob/48e90ed87e3116333f0b964a2eed352ca915a660/benchmark/common.js#L197
so calling:
```js
runBenchmark('zlib',
             [
               'method=deflate',
               'n=1',
               'options=true',
               'type=Deflate',
               'inputLen=1024',
               'duration=0.001'
             ],
             {
               'NODEJS_BENCHMARK_ZERO_ALLOWED': 1
             });
```
should get the test to pass.

Reply:
Thanks, done!